libnd4j/buildnativeoperations.sh

```diff
@@ -137,7 +137,8 @@ fi
 
 case "$OS" in
     linux-armhf)
-    export CMAKE_COMMAND="$CMAKE_COMMAND -D CMAKE_TOOLCHAIN_FILE=$HOME/raspberrypi/pi.cmake"
+    export RPI_BIN=$HOME/raspberrypi/tools/arm-bcm2708/arm-rpi-4.9.3-linux-gnueabihf/bin/arm-linux-gnueabihf
```

Comment:
Let's have users set say RPI_HOME here instead of HOME, and the same in the text below.